fedimint-client-legacy/src/lib.rs

```diff
@@ -133,6 +133,9 @@ pub struct GatewayClientConfig {
     /// `short_channel_id` when creating invoices to be settled by this
     /// gateway.
     pub mint_channel_id: u64,
+    /// Liquidity-based routing fee in millionths of a routed amount.
+    /// In other words, 10000 is 1%.
+    pub fees_proportional_millionths: u32,
```

Comment:
If you expect other fee structure systems, might be worthwhile to turn this into an enum, especially if gets written to database.

```rust
#[non_exhaustive]
pub enum FeeStructure {
  Proportional { per_million : u32 }
}
```

Reply:
Ack. I'm expecting this standard definition of [RoutingFees](https://github.com/lightningdevkit/rust-lightning/blob/main/lightning/src/routing/gossip.rs#L1057-L1065).

Reply:
> Ack. I'm expecting this standard definition of [RoutingFees](https://github.com/lightningdevkit/rust-lightning/blob/main/lightning/src/routing/gossip.rs#L1057-L1065).

@m1sterc001guy, @dpc,
I converted PR to draft while I attempt change with full RoutingFees definition